CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,4 @@
 - [BUG] Fix wrong spec date regarding release/0.12.0 (#750)
 - [HARDENING] Add check for null content-type in notifications (#743) 
 - [HARDENING] Add support for configurable number of partitions and replication factor in OrionKafkaSink (#736)
+- [HARDENING] Replace method getAccumulatorForRollBack with clone (#746)
```

Comment:
Please, specify the class, i.e.:

"Replace method Accumulator.getAccumulatorForRollback() with Accumulator.clone() (#746)"